common/web/input-processor/build.sh

```diff
@@ -23,20 +23,16 @@ builder_describe "Builds the standalone, headless form of Keyman Engine for Web'
   "@/common/web/keyman-version" \
   "@/common/web/keyboard-processor" \
   "@/common/predictive-text" \
+  "@/developer/src/kmc-model test:module" \
```

Comment:
```suggestion
  "@/developer/src/kmc-model test" \
```

The default name is 'project', not 'module', but no module should be needed anyway.